### PR TITLE
Commatrix workflow and step

### PR DESCRIPTION
**Pull request by danimjaffe**
Add relevant files and directories for commatrix workflow and step, that are required for the periodic job.

### CODE REVIEW

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-telco5g-commatrix

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-network-flow-matrix

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-network-flow-matrix

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-network-flow-matrix

File: ci-operator/step-registry/telco5g/commatrix/tests/telco5g-commatrix-tests-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - name: E2E_TESTS_CONFIG
+    default: ""
+    documentation: Additional configuration.
+  - name: T5CI_VERSION
+    default: ""
+    documentation: Version of OCP cluster to install.
+  - name: T5CI_JOB_TYPE
+    default: ""
+    documentation: Job type of Telco5G CI jobs
+  - name: TEST_RUN_FEATURES
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/commatrix/tests/telco5g-commatrix-tests-ref.yaml:23]: please remove TEST_RUN_FEATURES and E2E_TESTS_CONFIG

**Review by sshnaidm — CHANGES_REQUESTED** [review]

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-network-flow-matrix-bm periodic-ci-openshift-release-master-nightly-4.16-network-flow-matrix-single-node-bm

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml [hunk 2]
```diff
@@ -826,6 +826,28 @@ tests:
       enable:
       - observers-resource-watch
     workflow: telco5g-hcp-cnf
+- as: network-flow-matrix-bm
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml:829]: These should follow our naming scheme for CI jobs

https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/

Mostly I think you're missing platform and CNI provider.

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-metal-ovn-network-flow-matrix-single-node-bm periodic-ci-openshift-release-master-nightly-4.16-metal-ovn-network-flow-matrix-bm

**Comment by stbenjam** [comment]
Thank you, names look better 👍🏻 

/approve

**Pull request merged**